### PR TITLE
Support for Kotlin

### PR DESCRIPTION
**Pull request by nedimf**
Simple support for Kotlin

### CODE REVIEW

File: languages.go [hunk 1]
```diff
@@ -8,4 +8,5 @@ const (
 	NoLang    LangType = ""
 	GoLang             = "go"
 	SwiftLang          = "swift"
+	KotlinLang		   = "kotlin"
```
Comment (asilvr) [inline languages.go]: Can you recommit when using gofmt? The indentation is off.

File: cmd/main.go [hunk 2]
```diff
@@ -51,10 +51,13 @@ func main() {
 						dataset, _ := mystery.LoadDataset(mystery.SwiftLang)
 						ingredients := dataset.Generate("", flagImports)
 						mystery.PrintGenerate("Swift", ingredients)
+					case mystery.KotlinLang:
+						dataset, _ := mystery.LoadDataset(mystery.KotlinLang)
```
Comment (asilvr) [inline cmd/main.go:55]: Can you re-commit with gofmt here as well? Spacing is off

File: languages.go (hunk 1 above)
Comment (nedimf) [reply languages.go]: sure

File: cmd/main.go (hunk 2 above)
Comment (nedimf) [reply cmd/main.go:55]: sure

**Comment by asilvr** [comment]
Great PR! Thanks for adding a new language :)

**Comment by nedimf** [comment]
> Great PR! Thanks for adding a new language :)

Happy to help

**Pull request merged**